easybuild/framework/easyconfig/easyconfig.py

```diff
@@ -231,16 +229,17 @@ def parse(self):
             self.log.error("Specifications should be specified using a dictionary, got %s" % type(self.build_specs))
         self.log.debug("Obtained specs dict %s" % arg_specs)
 
-        parser = EasyConfigParser(self.path)
+        self.log.info("Parsing easyconfig file %s with rawcontent: %s" % (self.path, self.rawtxt))
+        parser = EasyConfigParser(filename=self.path, rawcontent=self.rawtxt)
```

Comment:
only if `self.path` is `None` is the rawcontent ever used in the `EasyConfigParser`? is this really intended?